libs/common/blob.hpp

```diff
@@ -74,15 +56,8 @@ namespace iroha {
      * Converts current blob to hex string.
      */
     std::string to_hexstring() const noexcept {
-      std::string res(size_ * 2, 0);
-      auto ptr = this->data();
-      for (uint32_t i = 0, k = 0; i < size_; i++) {
-        const auto front = (uint8_t)(ptr[i] & 0xF0) >> 4;
-        const auto back = (uint8_t)(ptr[i] & 0xF);
-        res[k++] = code[front];
-        res[k++] = code[back];
-      }
-      return res;
+      return bytestringToHexstring(std::string_view{
+          reinterpret_cast<const char *>(this->data()), this->size()});
```

Comment:
if you mean changing `bytestringToHexstring` to accept a `ByteRange` as argument, I'm afraid that this will lead to even more not-directly-related changes to this PR. I can leave a TODO in this case.